index.html

```diff
@@ -15,44 +15,9 @@ <h1 class="text-center">Components</h1>
             <hr class="line-md center" />
             <div class="container">
                 <h2>Link to components</h2>
-                <a href="avatar-badge.html">Avatar and Badge</a>
+                <a href="avatar-badge.html" class="btn bg-green rounded">Avatar and Badge</a>
+                <a href="alert-button.html" class="btn bg-green rounded">Alert and Button</a>
             </div>
-            <h2>Buttons</h2>
-            <hr class="line-sm" />
-            <article>
-                <h3>Shapes</h3>
-                <section class="container flex-container">
-                    <button class="btn">Sharp Edge</button>
-                    <button class="btn rounded-slight">Slight Round</button>
-                    <button class="btn rounded">Rounded Edges</button>
-                    <button class="btn capsule">Capsule</button>
-                </section>
-                <h3>Filled</h3>
-                <section class="container flex-container">
-                    <button class="btn bg-blue">Sharp Edge</button>
-                    <button class="btn bg-green">Sharp Edge</button>
-                    <button class="btn bg-red">Sharp Edge</button>
-                    <button class="btn bg-purple">Sharp Edge</button>
-                </section>
-                <h3>Border Only</h3>
-                <section class="container flex-container">
-					<button class="btn bd-blue">Sharp Edge</button>
-					<button class="btn bd-green">Sharp Edge</button>
-					<button class="btn bd-red">Sharp Edge</button>
-					<button class="btn bd-purple">Sharp Edge</button>
-				</section>
-                <h3>Disabled</h3>
-                <section class="container flex-container">
-                    <button class="btn" disabled>Sharp Edge</button>
-                </section>
-				</section>
```

Comment:
extra section tag.

Reply:
This code has been removed from this file. See the `-` sign on the right as well as the red color

Reply:
i completly missed that.

Reply:
Thanks for catching this error again